packages/jaeger-ui/src/components/TracePage/TracePageHeader/KeyboardShortcutsHelp.css

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 .KeyboardShortcutsHelp--cta {
   font-size: 1.2rem;
-  margin: 0 -7px;
+  margin: -3px -7px;
```

Comment:
This is to center the Keyboard shortcut panel icon vertically.

Reply:
Without this:
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/22408e12-c42c-4a14-83cc-a5c9ffa9dcf0)


With this:
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/bfcf0d4f-8b8e-40f1-b2e5-3b09220895e1)